samples/subsys/mgmt/osdp/sample.yaml

```diff
@@ -4,7 +4,7 @@ sample:
 tests:
   sample.mgmt.osdp:
     tags: osdp
-    filter: dt_compat_enabled_with_alias("gpio-leds", "led0") and CONFIG_SERIAL
+    filter: dt_compat_enabled_with_alias("gpio-leds", "led0") and dt_alias_exists("rng") and CONFIG_SERIAL
```

Comment:
Ok, this might work to exclude boards, but I don't see any board having any `rng` alias today:
`$ grep aliases -A 10 boards/*/*/*.dts | grep rng` doesn't return anything

Reply:
Sorry, my bad. At the boards level, I think we must filter by dt_chosen "zephyr,entropy".

Reply:
Can you check now?